doc/settings/gitlab.yml.md

```diff
@@ -4,7 +4,9 @@ Some of GitLab's features can be customized through
 [gitlab.yml][gitlab.yml.example] and [application.yml (GitLab
 CI)][application.yml.example]. If you want to change a `gitlab.yml` setting
 with omnibus-gitlab, you need to do so via `/etc/gitlab/gitlab.rb`. The
-translation works as follows.
+translation works as follows. For a complete list of available options, visit the
```

Comment:
@bbodenmiller What do you think about moving `The translation works as follows.` after the text you've added?